rules/call_peak.smk

```diff
@@ -33,7 +44,7 @@ rule genrich_pileup:
     computational footprint.
     """
     input:
-        get_genrich_replicates
+        unpack(get_genrich_replicates)
```

Comment:
in line 64 you call the input `{input.treps}`, so maybe call this `treps = unpack(get_genrich_replicates)` ?

Reply:
nvm, its a dict. im stupid